.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 default_stages: [push, manual]
 exclude: ^deprecated-dune-v1-abstractions/
 repos:
-- repo: https://github.com/offbi/pre-commit-dbt
+- repo: https://github.com/duneanalytics/pre-commit-dbt
```

Comment:
not sure this should be included?

Reply:
I think the extra commits that were merged into this branch are messing up with my actual changes, let me rebase :D

Reply:
Done